### ISSUE TITLE
Center the noRecentsText

### ISSUE THREAD

**Issue by rocboronat** [Fintasys/emoji_picker_flutter#19]
As only a `String` and `TextStyle` can be passed to the picker, there's no way to center the content of the no recent emojis page.

May you add a parameter for it? Or just use textAlign center. It will do the job in all cases I think.

Thanks!

![image](https://user-images.githubusercontent.com/456499/125069659-49febb80-e0b7-11eb-8fbb-1366f943ec47.png)

**Comment by Fintasys** [comment]
@rocboronat Thanks for reporting this issue ! I agree the text should be at least centered which I have done in this PR https://github.com/Fintasys/emoji_picker_flutter/pull/20 
<img width="332" alt="Screen Shot 2021-07-11 at 23 36 27" src="https://user-images.githubusercontent.com/11667234/125199694-585cfb00-e2a2-11eb-94ec-51e87a863b9d.png">
I haven't released it yet, just merged it into master.
You can use this until I gonna make another release.
```
  emoji_picker_flutter:
    git:
      url: https://github.com/Fintasys/emoji_picker_flutter.git
      ref: master
```
Let me know if that solves your issue for now 🙇

**Comment by rocboronat** [comment]
Thanks! I will! 😊😊😊

**Comment by rocboronat** [comment]
Hello! 👋 It works wonderfully. Thanks a lot, @Fintasys! Feel free to close the issue when you consider it right 😄

**Comment by Fintasys** [comment]
Released with `v1.0.7`
